scenarioo-server/src/main/java/org/scenarioo/model/configuration/Configuration.java

```diff
@@ -226,4 +240,13 @@ public void setCreateLastSuccessfulScenarioBuild(final boolean createLastSuccess
 		this.createLastSuccessfulScenarioBuild = createLastSuccessfulScenarioBuild;
 	}
 
+	public Color getDiffImageColor() {
+		//Required BigInteger. Because the hex value could be out of range (int is signed). With BigInteger we can avoid this issue.
+		int rgba = new BigInteger(diffImageColorRgbaHex.substring(2), 16).intValue();
```

Comment:
Is there no proper way to decode that (by also taking `0x` into account) - like for example Integer.decode is doing.